### ISSUE TITLE
Wrong handling of FourCCs with spaces at the end

### ISSUE THREAD

**Issue by mbunkus** [mbunkus/mtx-trac-import-test#107]
**Reported by moritz on 22 Mar 2010 12:00 UTC**
Reported by email:

On Windows XP/Vista (32 bit) I experience the following with MKVmerge
(V3.1.0) and mmg related to FOURCC handling and enforcing.
I have created an AVI file with full uncompressed frames using Virtualdub.
The AVI file has 4 character FOURCC "DIB " and works in a mediaplayer. I see
two phenomena related to MKVmerge, which I think are bugs.
1) When I convert the AVI file to MKV in MKVMerge or mmg, the FOURCC is not
generated in the MKV file automatically.
2) When I convert the AVI file to MKV in MKVMerge or mmg, and enforce a
FOURCC it complains about the FOURCC having to be 4 characters.
 
In short: the FOURCC handling appears to miss the trailing space character.
 
Example:
mkvmerge -f "0:DIB " -o "movie.mkv" "movie.avi"
mkvmerge v3.1.0 ('Happy up here') built on Jan 19 2010 12:09:24
Error: The FourCC must be exactly four characters long in '-f 0:DIB '.
 
If I change the space to some other character the FOURCC is accepted and the
merging starts.
Expected result: MKV file with FOURCC "DIB " (note the trailing space), both
by default when just converting the AVI, but also when the FOURCC is
enforced
